api4/bot.go

```diff
@@ -89,7 +90,8 @@ func patchBot(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	auditRec := c.MakeAuditRecord("patchBot", audit.Fail)
 	defer c.LogAuditRec(auditRec)
-	auditRec.AddMeta("bot_id", botUserId)
+	auditRec.AddEventParameter("bot_id", botUserId)
+	auditRec.AddEventParameter("bot_patch", botPatch)
```

Comment:
```suggestion
	auditRec.AddEventParameter("bot", botPatch)
```

As the event is called `Patch`, we can have a consistency across create and update events and use the same parameter. The question above re: `Auditable` remains.